tests/e2e/specs/in-game/one-offs/5_fives.spec.js

```diff
@@ -56,34 +50,40 @@ describe('FIVES', () => {
         // Deck
         topCard: Card.THREE_OF_CLUBS,
         secondCard: Card.EIGHT_OF_HEARTS,
-        deck: [],
+        deck: [Card.SEVEN_OF_CLUBS],
       });
-      cy.get('#deck').should('contain', '(2)');
+      cy.get('#deck').should('contain', '(3)');
 
       // Player plays and resolves a 5
       cy.playOneOffAndResolveAsPlayer(Card.FIVE_OF_CLUBS);
+      cy.get('[data-cy=five-discard-dialog]').should('be.visible');
+      cy.get('[data-cy=submit-five-dialog]').click();
+      
       assertGameState(0, {
-        // Player is P0
-        p0Hand: [Card.THREE_OF_CLUBS, Card.EIGHT_OF_HEARTS],
+        p0Hand: [Card.THREE_OF_CLUBS, Card.EIGHT_OF_HEARTS, Card.SEVEN_OF_CLUBS],
         p0Points: [],
         p0FaceCards: [],
-        // Opponent is P1
         p1Hand: [],
         p1Points: [],
         p1FaceCards: [],
         scrap: [Card.FIVE_OF_CLUBS],
+        topCard: null,
+        secondCard: null,
+        deck: []
       });
       // Deck should now be empty
       cy.get('#deck').should('contain', '(0)').should('contain', 'PASS');
+      cy.get('[data-player-hand-card]').should('have.length', 3);
     });
 
-    it('Plays a 5 to draw two cards when there are 3 cards in deck', () => {
+    it('Plays a 5 to draw two cards at max hand', () => {
```

Comment:
I think we additionally want a test case for drawing the last two cards of the deck